os/drivers/audio/alc5658.c

```diff
@@ -570,9 +592,22 @@ static int alc5658_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR stru
 			 */
 		}
 
+		switch (caps->ac_format.hw) {
```

Comment:
please check comment above regarding this. we should modify here and line 584~591

Reply:
Modified.